public/javascripts/script.js

```diff
@@ -1,31 +1,69 @@
 // This will print in the wrong order
 // we added it for you to test to make sure data is loaded
 // 🚨🚨🚨 comment out the next 3 lines when you start working on the code
-for (let i = 0; i < mashPotatoes.length; i++) {
-  //addFood(steak[i], '#steak');
-  console.log(mashPotatoes[i])
-}
+// for (let i = 0; i < mashPotatoes.length; i++) {
+//   //addFood(steak[i], '#steak');
+//   console.log(mashPotatoes[i])
+// }
 
 // Iteration 1 using callbacks
 addFood(steak[0], '#steak', () => {
-  // ... your code here
   addFood(steak[1], '#steak', () => {
-
-  })
+    addFood(steak[2], '#steak', () => {
+      addFood(steak[3], '#steak', () => {
+        addFood(steak[4], '#steak', () => {
+          addFood(steak[5], '#steak', () => {
+            addFood(steak[6], '#steak', () => {
+              addFood(steak[7], '#steak', () => {
+                let imgSteak = document.createElement('img');                 
```

Comment:
You could make it look clenar like this: 
document.querySelector('#table').innerHTML += `<img src="public/images/brusselSprouts.jpg"/>`
Nevertheless, good work!